content/ember-data/v5/ember-data-deprecate-computed-chains.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Deprecate Non Strict Types
+until: '6.0'
+since: '5.0'
+displayId: <none yet assigned>
+---
+
+This is a planned deprecation which will trigger when observer or computed chains are used to watch for changes on any EmberData RecordArray, ManyArray or PromiseManyArray.
```

Comment:
This deprecation should be written as if it has already happened and guess at the id.

Likely this deprecation also needs some before/after examples.

Reply:
This one shouldn't be here. We implement a guard for a specific behavior to allow folks to opt out of legacy chains, we don't print a deprecation because we've never figured out how to detect legacy chains (the work you do to support them is eager, not lazy, so can't rely on "pull" to trigger).

Reply:
removed this completely